packages/components/src/PlaceCard/PlaceCard.js

```diff
@@ -0,0 +1,112 @@
+// @flow
+
+import * as React from 'react';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { View } from 'react-native';
+import { Text, StyleSheet } from '@kiwicom/universal-components';
+
+import { Touchable } from '../Touchable';
+import { AdaptableBadge } from '../AdaptableBadge';
+// import BlackToAlpha from './assets/black-to-alpha-vertical.png'; @TODO
+import CityImage from './Image/CityImage';
+
+type Props = {|
+  +imageUrl?: string,
+  +price: string,
+  +place: string,
+  +country: string,
+  +onPress: () => void,
+  +disabled?: boolean,
```

Comment:
Not documented anywhere (yet) but isn't it nicer to keep mandatory props on top, and optional props at the bottom?

Reply:
It is a nice idea, I checked the `eslint-plugin-flowtype`, but I found only [sort-keys](https://github.com/gajus/eslint-plugin-flowtype#sort-keys) rule, which is not able to check this. 
So I don't see the big advantage from this if we can't ensure automatically check this rule.